### ISSUE TITLE
Resuming a push after a network interruption will systematically fail integrity check

### ISSUE THREAD

**Issue by henri-tremblay** [boxfuse/boxfuse-issues#213]
Each time I do a push that is interrupted because my computer goes to sleep or because the network connection breaks, it seems to resume when the connection is back.

However, it actually fails the integrity check systematically.

A quick fix would be to just fail instead of resuming. At least it's not misleading.

Then, of course, the real solution is to resume correctly.

**Comment by axelfontaine** [comment]
I fully agree. The correct way to resolve this would be to keep on retrying the parts with failed checksums until they succeed. In most cases it's just one 5MB part anyway, so that would be a lot faster.
